### DISCUSSION TITLE
How to debug errors connecting to workspace?

### DISCUSSION THREAD

**Discussion by mserranom** [community/community#6543] [Codespaces]
I'm unable to connect to my workspace. The image is built succesfully, but it gets stuck connecting after ~1m, ends up showing a `VSCodeWorkbench mount failed with "Failed to start VS Code Remote server."` sign in the browser.

I'm also unable to debug in VSCode, it simply shows "Failed to start VS Code Remote server."

Is there a way to debug what's going on behind the scenes?

<img width="575" alt="Screenshot 2021-10-20 at 07 12 54" src="https://user-images.githubusercontent.com/5948929/138032584-47cd9c02-adf7-4cfa-acd8-f4e5081cd27b.png">

<img width="569" alt="Screenshot 2021-10-20 at 07 15 38" src="https://user-images.githubusercontent.com/5948929/138032590-71fc5cb4-0940-4d43-ad77-e174bfa49e72.png">

**Comment by matthewisabel** [comment]
Hi @mserranom, I unfortunately don't think we have any logs here that can be extracted but we can try to debug on our side. Is this happening repeatedly and is it reproducing across browser and VS Code desktop?

**Comment by jahilliard** [comment]
The issue is happening for me too.  Is there a standard fix for this?  Or does this need to investigated internally?
